dreadpool/src/test/java/org/comroid/test/dreadpool/FixedSizeThreadPoolTest.java

```diff
@@ -106,6 +106,7 @@ public void test() throws InterruptedException {
         UUID[] arr2 = reversed1.toArray(new UUID[0]); // reversed
         for (int i = 0; i < arr2.length; i++)
             assertEquals("index: " + i, someTasks.get(i + 10).uuid, arr2[i]);
+        // todo: fails probably due to a concurrency problem
```

Comment:
Then test it. 
You can create concurrency tests which help debugging and creating a thread safe pool.

Reply:
that todo isnt accurate, i'M just too stupid to reverse collections obviously. somethings wrong in the test bcs idk junit lol